src/styles/GlobalStyle.js

```diff
@@ -0,0 +1,85 @@
+import { createGlobalStyle } from "styled-components"
+import { themeColors } from "./colors"
+import { CSSReset } from "./CSSReset"
+import { font } from "./fonts"
+
+export const GlobalStyles = createGlobalStyle`
+  ${CSSReset};
+  html, body {
+    color: black;
+    font-family: ${font};
+    background-color: ${themeColors.primary1};
+  }
+
+  ::-webkit-scrollbar {
```

Comment:
can you delete the `CSS` rules for `::-webkit-scrollbar` & `::-webkit-scrollbar-thumb`, these styles will change the way the scroll bar looks, which we don't want on the marketing page, forgot to delete them when giving them to you.